library/src/main/java/com/github/paolorotolo/appintro/AppIntro2Fragment.java

```diff
@@ -12,15 +12,34 @@ public static AppIntroFragment newInstance(CharSequence title, CharSequence desc
         AppIntroFragment slide = new AppIntroFragment();
         Bundle args = new Bundle();
         args.putString(ARG_TITLE, title.toString());
+        args.putString(ARG_TITLE_TYPEFACE,"");
         args.putString(ARG_DESC, description.toString());
+        args.putString(ARG_DESC_TYPEFACE,"");
         args.putInt(ARG_DRAWABLE, imageDrawable);
         args.putInt(ARG_BG_COLOR, bgColor);
         args.putInt(ARG_TITLE_COLOR, titleColor);
         args.putInt(ARG_DESC_COLOR, descColor);
         slide.setArguments(args);
         return slide;
     }
+    public static AppIntroFragment newInstance(CharSequence title, String titleTypeface, CharSequence description, String descTypeface, int imageDrawable, int bgColor) {
+        return newInstance(title, titleTypeface ,description, descTypeface, imageDrawable, bgColor, 0, 0);
```

Comment:
There is a stray comma here, go ahead and rewrite:
`return newInstance(title, titleTypeface, description, descTypeface, imageDrawable, bgColor, 0, 0);`